src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/ErrorCodes.scala

```diff
@@ -10,4 +10,5 @@ object ErrorCodes {
   val notAuthorised = "NOT_AUTHORISED"
   val invalidConsignmentState = "INVALID_CONSIGNMENT_STATE"
   val invalidInputData = "INVALID_INPUT_DATA"
+  val unknownError = "UNKNOWN_ERROR"
```

Comment:
This doesn't look like it's used any more.

Reply:
Good spot, I've removed it.